v2/internal/platform/win32/winres.go

```diff
@@ -0,0 +1,7 @@
+package win32
+
+import "unsafe"
+
+func MakeIntResource(id uint16) *uint16 {
```

Comment:
I think using this one will make problems with the race detector as seen in: #1554